index.html

```diff
@@ -5,30 +5,6 @@
     <title>Multicapture</title>
     <script class="remove" src="multicapture.js" type="text/javascript"></script>
     <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove"></script>
-    <script class="remove">
```

Comment:
It might be better to keep the inline config and drop the JS based one

Reply:
Done.